xchange-coinbaseex/src/test/java/com/xeiam/xchange/coinbaseex/CoinbaseExExchangeTest.java

```diff
@@ -1,12 +1,29 @@
 package com.xeiam.xchange.coinbaseex;
```

Comment:
I don't think this should be a unit test because if Coinbase changes the currency pairs, the test will fail. At the least, it should be an integration test. Preferably, it should be move to `xchange-examples` as a demo class.